docker/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM alpine:3.8 as release
+
+RUN apk --no-cache add ca-certificates \
+  && apk --no-cache add \
+    curl \
+    tar
+
+RUN mkdir /source
+WORKDIR /source
+RUN curl -Lo strichliste.tar.gz https://github.com/strichliste/server/releases/download/v0.2.3/strichliste.tar.gz
+RUN tar -xf strichliste.tar.gz
+RUN rm -r strichliste.tar.gz
+
+FROM alpine:3.8
+
+RUN apk --no-cache add ca-certificates \
+  && apk --no-cache add \
+    curl \
+    php7 \
+    php7-ctype \
+    php7-tokenizer \
+    php7-iconv \
+    php7-mbstring \
+    php7-xml \
+    php7-json \
+    php7-dom \
+    php7-pdo_sqlite \
+    php7-pdo_mysql \
+    php7-fpm \
+    nginx
+
+COPY --from=release /source source
+
+COPY entrypoint.sh /source/entrypoint.sh
+RUN chmod +x /source/entrypoint.sh
+
+RUN adduser -u 82 -D -S -G www-data www-data
```

Comment:
Maybe we should use a environment variable to supply the `uid` and `gid`? That simplifies migrations and backups.

Reply:
I usually provide default values in the Dockerfile. If you create the user in the entrypoint script there should not be any pitfalls.

```
ENV UID=1000 \
    GID=1000
```

Reply:
Both `useradd` and `chown` need root. So using `USER www-data` wouldn't be possible and the container starts and runs as root. That's the primary argument against setting them as ENVs in my view.